msfvenom

```diff
@@ -69,7 +69,7 @@ def parse_args(args)
   opt.separator('')
   opt.separator('Options:')
 
-  opt.on('-l', '--list            <type>', Array, 'List all modules for [type]. Types are: payloads, encoders, nops, platforms, encrypt, formats, all') do |l|
+  opt.on('-l', '--list            <type>', Array, 'List all modules for [type]. Types are: payloads, encoders, nops, platforms, arches, encrypt, formats, all') do |l|
```

Comment:
Not a big fan of the plural here, [`archs`](https://writingexplained.org/english-abbreviations/architecture)? Since we using the `'` would likely be bad idea (see link).

Reply:
Yeah, I don't actually like any pluralization of the abbreviation. I considered just "arch." Thoughts? "Archit" is just ridiculous in a computing context.

Reply:
I also don't care for the period in this context. I think I can get behind `archs` here, since it's the closest to our singular representation. I'll make that change, thanks.

Reply:
In retrospect, I think this was just a brain fart on my part while writing it out. "Archs" it is. It's the most correct given our abbreviated form, and it can be pronounced with a hard C as per "architecture."

Reply:
I most definitely pronounce it "arches," though. Haha. English sucks.

Reply:
😆 my first request was almost remove plural from all items, I figured that would get rejected so this was my compromise.

Reply:
Damn, I would have gone for that.